src/main/java/thePackmaster/hats/HatMenu.java

```diff
@@ -53,25 +53,26 @@ public class HatMenu {
     private static final TextureRegion MENU_BG = new TextureRegion(ImageMaster.loadImage("img/ModPanelBg.png"));
 
     //positions
-    private static final float BG_X_SCALE = Settings.scale * 0.275f;
-    private static final float BG_Y_SCALE = Settings.scale * 0.8f;
-    private static final float BG_X = 525f * Settings.scale;
-    private static final float BG_Y = Settings.HEIGHT - 40f * Settings.scale - MENU_BG.getRegionHeight() * BG_Y_SCALE;
+    private static final float BG_X_SCALE = Settings.xScale * 0.275f;
```

Comment:
Most of the changes are just like this -- take something using `Settings.scale`, switch it to using whichever of `Settings.xScale` or `Settings.yScale` is appropriate, problem solved.